packages/daemon/src/services/index.ts

```diff
@@ -238,7 +239,7 @@ export const handleVertexAccepted = async (context: Context, _event: Event) => {
       const blockRewardOutput = outputs[0];
 
       // add miner to the miners table
-      if (blockRewardOutput.decoded) {
+      if (isDecodedValid(blockRewardOutput.decoded)) {
```

Comment:
`decoded.address` is required here, right?

Reply:
Done, thanks